src/components/interactive-builder/edit-question-modal.component.tsx

```diff
@@ -432,7 +432,7 @@ const EditQuestionModal: React.FC<EditQuestionModalProps> = ({
                     kind="error"
                     lowContrast
                     className={styles.error}
-                    title={t('errorFetchingPersonAttributeTypes', 'Error fetching  person attribute types')}
+                    title={t('errorFetchingPersonAttributeTypes', 'Error fetching person attribute types')}
```

Comment:
Looks like there were some formatting issues in this file. The changes I made here were to remove some unnecessary spaces in two translation strings.